addons/portal/static/src/js/portal_chatter.js

```diff
@@ -20,7 +21,8 @@ var PortalChatter = publicWidget.Widget.extend({
     template: 'portal.Chatter',
     xmlDependencies: ['/portal/static/src/xml/portal_chatter.xml'],
     events: {
-        "click .o_portal_chatter_pager_btn": '_onClickPager',
+        'click .o_portal_chatter_pager_btn': '_onClickPager',
+        'click .o_portal_js_internal_toggle': '_onClickToggleInternal',
```

Comment:
'async _onClickToggleInternal' to avoid double click issues

Reply:
Good idea.